### ISSUE TITLE
Attempting to use Realm-recyclerview with Object Server....

### ISSUE THREAD

**Issue by ericwhiteau** [thorbenprimke/realm-recyclerview#98]
I am attempting to implement your realm Recyclerview with a dataset that will be central to several users, hence use of ROS. 

When I attempt to add the recyclerview I get the following error.
The StackOverflow discussions seem to reference where you have different versions of realm and ROS.  this is not the case here.

I have also tried Packaging Options
eg.
/        pickFirst 'lib/x86/librealm-jni.so'
//        pickFirst 'lib/x86_64/librealm-jni.so'
//        pickFirst 'lib/arm64-v8a/librealm-jni.so'

Pack options allows compile, however I then get a runtime fail not connecting to the server.

error below after adding the Realm Recycler library.
-----------
Error:Execution failed for task ':app:transformNative_libsWithMergeJniLibsForDebug'.
> com.android.build.api.transform.TransformException: com.android.builder.packaging.DuplicateFileException: Duplicate files copied in APK lib/x86/librealm-jni.so
	File1: C:\data\src\Realm\ResultList\app\build\intermediates\exploded-aar\io.realm\realm-android-library\2.2.1\jni
	File2: C:\data\src\Realm\ResultList\app\build\intermediates\exploded-aar\io.realm\realm-android-library-object-server\2.2.1\jni

**Comment by ericwhiteau** [comment]
Just to add, I understand I am getting a Dup library problem, however, I cant find how to fix it.

I have run the example code from your tutorial with success, but that didnt use the server.  I have not tried to run the example on ROS.

cheers and thanks in advance.

**Comment by ericwhiteau** [comment]
I tried adding the Realm Server to your example code.  this resulted in the same error message.

I have now copied manually each class into a realmUtils package in my project.  This compiles and runs.

However, I have some issues with  realmResults no being allowd across package.   see below
realmResults.getTableOrView().getTable().getColumnIndex(headerColumnName);  

and some others.  e.g. BaseRealm with swipeDismiss.

anyway I will push ahead trying to isolate it.  

any thoughts or pointers would be appreciated.
